docs/workforce-management-framework/topics/con-raincatcher-core-getting-started.adoc

```diff
@@ -0,0 +1,143 @@
+[[id='raincatcher-core-getting-started-{chapter}']]
+= RainCatcher-core Getting Started
+
+Getting started with the Feedhenry RainCatcher Core Repository
+
+== Developing
+
+RainCatcher supports and is tested on Node.JS@4 and should run on latest LTS.
+
+Check your Node.JS version running `node -v`
+
+=== Setup
+
+[source,bash]
+----
+    git clone https://github.com/feedhenry-raincatcher/raincatcher-core.git
+    cd raincatcher-core
+    npm install -g lerna
+    npm install
+    lerna bootstrap
+----
+
+== Repository folder structure
+
+This repository contains many subpackages managed through https://lernajs.io/[Lerna],
+and they are contained in the following directories:
+
+* *client/*
+- Front-end packages, from API clients to user interface modules
+* *cloud/*
+- Cloud app packages, intended to be run on an Node.JS environment
+* *demo/*
+- Full-fledged Demo applications, showcasing the usage of multiple modules
+* *templates/*
+- Templates and examples for other packages in the repository
+
+
+== Creating a new package
+
+In order to create a new package, we recommend duplicating
+`templates/base`, which is a sample base that contains the skeleton expected of a new package.
+Review the `/templates/base/README.md` for further information.
+
+=== Package structure
+
+==== Directories
```

Comment:
I believe this heading pertains to the directories of WFM, not this repo. If I am correct, this needs to be extracted from here and put into a "ref-raincatcher-core-getting-started.adoc" as it's more "factual" (reference) information rather than concept info. If I am wrong, just pull it out as it is most likely duplicate work(prob best to confirm).